service/index.js

```diff
@@ -0,0 +1,38 @@
+const Contact = require("./schemas/contactSchemas");
+
+const getAllContacts = () => {
+  return Contact.find();
+};
+
+const getContactById = (id) => {
+  if (id.match(/^[0-9a-fA-F]{24}$/)) {
```

Comment:
naprawde tego potrzebujesz? Skad ten pomysl?

Reply:
Jest to sprawdzenie czy id jest prawidłowym ObjectId w Moongose. Znalazłem to na stackoverflow. Bez tego zapisu w przypadku nie znalezienia szukanego id pojawia się błąd:
CastError: Cast to ObjectId failed for value "6362e4a5949d0967b083a77" (type string) at path "_id" for model "contact". I zamiast wyłapywać błąd 404 z kontrolera wyłapuje błąd 500 z servera.js tj: "Internal Server Error". Kiedy id jest znalezione, wszystko okey, objectId jest wtedy prawidłowe.

Reply:
Ciekawe... 🤔 A myslisz, ze moglbys zadbac o to prosciej? np. poprzez `mongoose.ObjectId(id)`, zeby `id` (zamiast Stringiem) bylo ObjectId?